src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,131 @@
-import { PeopleFilters } from './PeopleFilters';
+// import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { PeopleFilters } from './PeopleFilters';
+import React, { useEffect, useState } from 'react';
+import { sortFunction } from '../utils/sorter';
+import { useSearchParams } from 'react-router-dom';
 
 export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [fetchError, setFetchError] = useState(false);
+  const [showLoader, setShowLoader] = useState(false);
+  const [renderedPeople, setRenderedPeople] = useState<Person[]>([]);
```

Comment:
why do you need two states for people, try to rewrite to use only one
`const renderedPeople = getRenderedPeople(......)`

Reply:
I need "people" and "renderedPeople" because I use it for different purposes.
"people" it's always our fetched array that we pass to our filters, then we sort this filtered people and this IS our renderedPeople. (you can see this process in useEffect in PeoplePage component)
so I always need some basic array of people